### PR TITLE
[TASK] Rework Windows setup instructions (#228)

### PR DESCRIPTION
**Pull request by liayn**

### CODE REVIEW

File: Documentation/Appendix/Windows/GitForWindows.rst [hunk 1]
```diff
@@ -13,204 +13,69 @@ Working with Git for Windows
 Installation
 ============
 
-Download Git For Windows from  https://git-scm.com/.
+Download Git For Windows from  https://git-scm.com/download/win.
```
Comment (susannemoog) [inline Documentation/Appendix/Windows/GitForWindows.rst:16]: should we at least add a note and mention wsl2 as an alternative?

Comment (liayn) [reply Documentation/Appendix/Windows/GitForWindows.rst:16]: Of course! good point

File: Documentation/Appendix/Windows/SSHGitWindows.rst [hunk 2]
```diff
@@ -79,11 +79,11 @@ Better make sure your editor is set to use LF while contributing to TYPO3.
 
 .. image:: /Images/External/GitSetup/git-setup-9.jpg
 
-Select the terminal of your choice. MinTTY is a good choice, because it's mostly what you would exect from a terminal.
+Select the terminal of your choice. MinTTY is a good choice, because it's mostly what you would expect from a terminal.
 
 .. image:: /Images/External/GitSetup/git-setup-10.jpg
 
-Choosing "Rebase" is a rather save default setting. It makes sure that your commits are placed on top of whatever you pull in from the remote.
+Choosing "Rebase" is recommended. It makes sure that your commits are placed on top of whatever you pull in from the remote.
```
Comment (liayn) [inline Documentation/Appendix/Windows/SSHGitWindows.rst:86]: @tomwarwick Thanks for the updates Tom!
In regards to this point, I'm not sure if we should use "recommended" here. It is for sure recommended for TYPO3 repositories, but here we talk global settings.
That's why I went with "safe default setting" (incl a typo ;) ).

Comment (tomwarwick) [reply Documentation/Appendix/Windows/SSHGitWindows.rst:86]: No problem, I'll change it back - thanks for the feedback :)

**Review by linawolf — APPROVED** [review]

**Comment by sypets** [comment]
Thank you @liayn  and @linawolf

**Pull request merged**